src/nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py

```diff
@@ -287,10 +293,17 @@ def run_conversion(
                         chunks=best_gzip_chunk,
                     )
                 image_series_kwargs.update(data=data)
-                if check_regular_series(series=timestamps):
-                    image_series_kwargs.update(starting_time=starting_times[j], rate=fps)
-                else:
-                    image_series_kwargs.update(timestamps=timestamps)
+
+            is_regular, rate = check_regular_series(series=timestamps)
+            if is_regular:
+                if fps != rate:
+                    warn(
+                        f"The fps={fps} from movie data is unequal to the difference in"
+                        f"regular timestamps. Using fps={rate} from timestamps instead."
+                        , UserWarning)
```

Comment:
@CodyCBakerPhD I'm not sure why but this warning doesn't get triggered, even when i'm in debug mode and put a breakpoint at this line it gets there, but no warning is triggered -- the unittest is failing currently. Do you have any idea why?

Reply:
@weiglszonja See comments below - the context must encapsulate the function call that actually gives the warning